packages/interpretations/src/components/interpretations/CommentTextarea.js

```diff
@@ -3,31 +3,42 @@ import PropTypes from 'prop-types'
 import { Link, ActionSeparator } from './misc';
 import i18n from '@dhis2/d2-i18n'
 import styles from './InterpretationsStyles.js';
-import RichEditor from '../html-editor/RichEditor';
 
 class CommentTextarea extends React.Component {
     static propTypes = {
         comment: PropTypes.object.isRequired,
         onPost: PropTypes.func.isRequired,
         onCancel: PropTypes.func,
-        mentions: PropTypes.object,
     };
 
     constructor(props) {
         super(props);
-        this.state = { text: props.comment.text || "", refresh: new Date() };
+        this.state = { text: props.comment.text || "" };
         this.onPost = this.onPost.bind(this);
```

Comment:
Should probably remove the bind functions and convert to class properties instead to avoid boilerplate code. 

State can also be moved out of the constructor so that the entire constructor can be deleted.